### PR TITLE
Create Alpaca Tips Docs

### PR DESCRIPTION
**Pull request by elizoller**
part of documentation sprint

# What does this Pull Request do?
Adds a page about Alpaca and how to make (basic) changes in the playbook

# What's new?
A new markdown page

# How should this be tested?

- Read it. 
- Fill in things.
- Correct things.

# Additional Questions
Not sure how this fits in with http://islandora.github.io/documentation/alpaca/alpaca-technical-stack/

# Interested parties
 @Islandora-CLAW/committers

### CODE REVIEW

File: docs/technical-documentation/alpaca_tips.md
Comment (seth-shaw-unlv) [inline docs/technical-documentation/alpaca_tips.md]: typo: 'Apachel' -> 'Apache'

File: docs/technical-documentation/alpaca_tips.md
Comment (seth-shaw-unlv) [inline docs/technical-documentation/alpaca_tips.md]: Since you describe each of these in turn (well, at least make sections for each of them) you can link them:
```
- [islandora-connector-derivative](#islandora-connector-derivative)
```

**Comment by seth-shaw-unlv** [comment]
Other than the typo (and, optionally, the section links) this looks good to me. I used the instructions to test the versioning PR. That said, I would feel better about merging if someone with more Alpaca experience than me (@whikloj, perhaps?) looked it over, too.

**Review by seth-shaw-unlv — APPROVED** [review]
As I said before, looks good to me, but I'll hold off merging to give others a chance to look it over.

**Comment by elizoller** [comment]
thanks @seth-shaw-unlv

File: docs/technical-documentation/alpaca_tips.md
Comment (dannylamb) [inline docs/technical-documentation/alpaca_tips.md:14]: `islandora-http-client` overrides the default http client with Islandora specific configuration.

File: docs/technical-documentation/alpaca_tips.md
Comment (dannylamb) [inline docs/technical-documentation/alpaca_tips.md]: You could shrink that first sentence to "This service receives requests from Drupal in response to write operations on entities."

**Review by dannylamb — CHANGES_REQUESTED** [review]
Left a few things.  But overall looks good.  That's pretty much what I do when re-deploying specific bundles.

**Comment by dannylamb** [comment]
@elizoller http://islandora.github.io/documentation/alpaca/alpaca-technical-stack/ is outdated and needs :eyes:.  It's not even in the TOC. So I wouldn't worry about it for now, and when someone goes over it, we can combine where appropriate.

File: docs/technical-documentation/alpaca_tips.md
Comment (elizoller) [reply docs/technical-documentation/alpaca_tips.md:14]: oh, thanks!

**Review by dannylamb — APPROVED** [review]

**Comment by dannylamb** [comment]
@elizoller Looks good.  Want to add this to the table of contents in the mkdocs file?

**Review by dannylamb — APPROVED** [review]

**Pull request merged**
